pkg/tenant/apikeys.go

```diff
@@ -82,7 +82,7 @@ func (s *Server) ProjectAPIKeyList(c *gin.Context) {
 	var reply *qd.APIKeyList
 	if reply, err = s.quarterdeck.APIKeyList(ctx, req); err != nil {
 		log.Error().Err(err).Msg("could not list API keys")
-		c.JSON(qd.ErrorStatus(err), api.ErrorResponse("could not list API keys"))
+		api.ReplyQuarterdeckError(c, err)
```

Comment:
We should create a follow on story to make sure that Quarterdeck is returning good error responses to the user (e.g. if it's an internal error it should be pretty generic).